### PR TITLE
Logo on header and replaced the favicons

### PR DESCRIPTION
**Pull request by renatarko**
I worked on this: https://github.com/ocodista/trampar-de-casa/issues/62

- [x] I updated the logo on header and I used the component Image from next to improve image optimization;
- [x] Add name and short_name on manifest file;
- [x] And I replace the favicons files.

image

### CODE REVIEW

File: apps/web/app/landing-page/Hero.tsx [hunk 1]
```diff
@@ -98,9 +98,10 @@ export const Hero = () => {
               <div className="flex flex-wrap items-center">
                 <div className="w-auto mr-14">
                   <a href="#">
-                    <img
-                      src="images/casa.png"
-                      className="h-[80px]"
+                    <Image
+                      width={100}
+                      height={100}
```
Comment (ocodista) [inline apps/web/app/landing-page/Hero.tsx:103]: I think it would be better if the logo was shorter.

File: apps/web/app/landing-page/Hero.tsx [hunk 2]
```diff
@@ -40,15 +41,14 @@ export const Hero = () => {
 
   useEffect(() => {
     (async () => {
-      const response = await fetch(ApiRoutes.Subscribers)
+      const response = await fetch(ApiRoutes.Subscribers);
       if (response.ok) {
-        const count = await response.json()
-        setSubscribersCount(count)
```
Comment (ocodista) [inline apps/web/app/landing-page/Hero.tsx:46]: It looks like your linting is adding ";" to the end of the lines, would you please undo these changes and check only what is indeed change?

Thank you in advance!

**Review by ocodista — CHANGES_REQUESTED** [review]

**Comment by renatarko** [comment]
I made the changes on commit: https://github.com/ocodista/trampar-de-casa/pull/63/commits/61965921610abfd33ec30256c4b68b95c7f95317

And about the add ';' I would like to know with is missing some settings on my project, because I'm running the yarn format and I'm getting a lot of changes:

image

**Comment by ocodista** [comment]
> I made the changes on commit: [6196592](https://github.com/ocodista/trampar-de-casa/commit/61965921610abfd33ec30256c4b68b95c7f95317)
> 
> And about the add ';' I would like to know with is missing some settings on my project, because I'm running the yarn format and I'm getting a lot of changes:
> 
> image

You're right, there is a problem with the prettier/eslint config.

I opened #66 as I won't be able to work on it right now.

Your PR looks good, but I'll wait for the format/linting issue to be fixed.

Also, is there a way to use another color for the favicon? In my opinion it get's a little ofuscated.

**Comment by renatarko** [comment]
For the favicon I decided to create a derivative logo, what do you think?

image

T (2)

or keep the same logo but with a background to improve visualization

1 - 
2 -

File: apps/web/app/landing-page/Hero.tsx (hunk 2 above)
Comment (tgmarinho) [reply apps/web/app/landing-page/Hero.tsx:46]: I saw there is PR about eslint/prettier config, it's an issue in this setting. I realized that some files are using `;`

**Comment by tgmarinho** [comment]
good work 🚀

File: apps/web/app/landing-page/Hero.tsx (hunk 2 above)
Comment (tgmarinho) [reply apps/web/app/landing-page/Hero.tsx:46]: https://github.com/ocodista/trampar-de-casa/pull/63#issuecomment-1616167018

**Comment by ocodista** [comment]
> For the favicon I decided to create a derivative logo, what do you think?
> 
> image
> 
> T (2)
> 
> or keep the same logo but with a background to improve visualization
> 
> 1 -  2 - 

Hello Renata, we can use the one you created for now, thank you!

**Review by ocodista — APPROVED** [review]
You're good to go with the current new logo and favicon.

Thank you for all the hard work!

File: apps/web/app/landing-page/Hero.tsx [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+});
+
+type ValidationSchema = z.infer<typeof validationSchema>;
+
+const PADDING_X = 32; 
+
+
+
+export const Hero = () => {
+ const { isLoading, withLoading } = useContext(LoadingContext);
```
Comment (ocodista) [inline apps/web/app/landing-page/Hero.tsx:27]:
```suggestion
  const { isLoading, withLoading } = useContext(LoadingContext);
```
